internal/wrappers/results-sbom-http.go

```diff
@@ -0,0 +1,129 @@
+package wrappers
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+
+	commonParams "github.com/checkmarx/ast-cli/internal/params"
+	"github.com/spf13/viper"
+
+	"github.com/pkg/errors"
+)
+
+type SbomReportsPayload struct {
+	ScanID     string `json:"ScanId"`
+	FileFormat string `json:"FileFormat"`
+}
+
+type SbomReportsResponse struct {
+	ExportID string `json:"exportId"`
+}
+
+type SbomPoolingResponse struct {
+	ExportID     string `json:"exportId"`
+	ExportStatus string `json:"exportStatus"`
+	FileURL      string `json:"fileUrl"`
+}
+type SbomHTTPWrapper struct {
+	path string
+}
+
+func NewResultsSbomReportsHTTPWrapper(path string) ResultsSbomWrapper {
+	return &SbomHTTPWrapper{
+		path: path,
+	}
+}
+
+func (r *SbomHTTPWrapper) GenerateSbomReport(payload *SbomReportsPayload) (*SbomReportsResponse, *WebError, error) {
```

Comment:
we are always returning WebError as nil. does the API return a body in failure codes? if yes, decode it, otherwise remove the return

Reply:
removing the WebError :) 
Thx!